### DISCUSSION TITLE
Podemos imprimir valores en las funciones?

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#78] [Preguntas y respuestas] [answered]
Podemos imprimir valores en las funciones?

**Comment by martinvilu** [comment, marked as answer]
Depende mucho de que función/ejercicio.

Para ver _qué onda_ con el programa si, idealmente con un comentario que lo aclare.

En qué ejercicio estás pensando?

**Reply by rinrot** [reply]
estaba pensando en el ejercicio 3 del Tp2 despues de que un compañero me aviso sobre no imprimir dentro de las funciones, y como ya entregue el tp ayer ya me estaba por poner a corregirlo jajaja

**Reply by martinvilu** [reply]
En el ejercicio de la tabla de multiplicar, no hay otra opción. Probablemente sea el único tambien.
